index.go

```diff
@@ -226,9 +231,9 @@ func main() {
 		r.Get("/user/:id", customer_ctlr.GetUserById)
 		r.Post("/user/:id", customer_ctlr.UpdateCustomerUser)
```

Comment:
I'll check in the logs, but almost everything seems to just use one key, only difference is that the dealer site seems to use the customers key when doing requests.